zqd/handlers_test.go

```diff
@@ -45,6 +45,27 @@ func TestSearch(t *testing.T) {
 	require.Equal(t, test.Trim(src), res)
 }
 
+func TestSearchReverse(t *testing.T) {
```

Comment:
nit: name it AggregateReverse or GroupbyReverse or something?
Not necessary though, feel free to merge as-is if this has gone on long enough :)

Reply:
done